orttraining/orttraining/eager/opgen/opgen/atenops.py

```diff
@@ -115,6 +102,19 @@ def __init__(self, dY, X):
 for unary_op in unary_ops:
     ops[f"aten::{unary_op}"] = onnx_ops[unary_op]("self")
 
+for binary_op, onnx_op in {
+    "add": Add("self", Mul("alpha", "other")),
+    "sub": Sub("self", Mul("alpha", "other")),
+    "mul": Mul("self", "other"),
+    "div": Div("self", "other"),
+}.items():
+    # for Tensor, binary_op.out is used by both binary_op and binary_op_, so we only generate .out
+    # from testing and call stacks, it also apears scalar ops fall back to the binary_op.out, so this is all we need.
+    name = f"aten::{binary_op}.out"
+    if name not in ops:
```

Comment:
This line (I know it's not new) seems like a bug waiting to happen.
At this point in processing, ops should only have unary ops in it.  If one of the binary ops is already in ops, that seems like a problem, not something to be skipped over.  In a follow up, consider if this should either throw or be removed.